docs/src/formulation.md

```diff
@@ -424,6 +408,28 @@ v^{\text{inv}}_{f} \leq \frac{p^{\text{inv limit}}_{f}}{p^{\text{capacity}}_{f}}
 
 If the parameter `investment_integer` in the [`flows-data.csv`](@ref flows-data) file is set to true, then the right-hand side of this constraint uses a least integer function (floor function) to guarantee that the limit is integer.
```

Comment:
I'm a bit worried this also crosses the line between Explanation and Reference/How-To.

Reply:
We don't have a how-to section for this, so we are leaving this here like this for now. Let's improve it later in the whole review of the documentation. Thanks!